plugins/user-settings/src/components/ThemeToggle.tsx

```diff
@@ -54,6 +47,36 @@ export const SidebarThemeToggle = () => {
     }
   };
 
+  // ToggleButtonGroup uses React.children.map instead of context
+  // so wrapping with Tooltip breaks ToggleButton functionality.
+  const TooltipToggleButton = ({
+    children,
+    title,
+    value,
+    ...props
+  }: {
+    children: JSX.Element;
+    title: string;
+    value: string;
+  }) => (
+    <Tooltip placement="top" arrow title={title}>
+      <ToggleButton value={value} {...props}>
+        {children}
+      </ToggleButton>
+    </Tooltip>
+  );
+
+  const ThemeIcon = ({ theme }: { theme: AppTheme }) => {
```

Comment:
Hm are these components defined and created inside another component? Feels like they should be outside at the root level to make it easier to reason about them and not have them be rebuilt and rerendered every time